From: Peter Wu <peter@lekensteyn.nl>
Date: Thu, 30 Oct 2014 10:22:07 +0100
Subject: Ignore systemd-logind noise

This "error" appears when not run under logind. As it is not fatal,
ignore it. (It also appears on my KDE desktop using KDM.)

Error message reported in bugs GH-600, GH-582.
---
 src/bblogger.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/bblogger.c b/src/bblogger.c
index aaadba9..c882275 100644
--- a/src/bblogger.c
+++ b/src/bblogger.c
@@ -145,6 +145,7 @@ static void parse_xorg_output(char * string){
   if (strncmp(string, "(EE)", 4) == 0){
     if (strstr(string, "Failed to load module \"kbd\"") ||
             strstr(string, "No input driver matching") ||
+            strstr(string, "systemd-logind: failed to get session:") ||
             strstr(string, "Server terminated successfully")) {
       /* non-fatal errors */
       prio = LOG_DEBUG;
@@ -201,7 +202,7 @@ static void parse_xorg_output(char * string){
       }
     }
   }
-  
+
   /* do the actual logging */
   bb_log(prio, "[XORG] %s\n", string);
 }
